src/repl/session.rs

```diff
@@ -1202,6 +1217,12 @@ impl Session {
         output.append(&mut result);
     }
 
+    #[cfg(not(feature = "cli"))]
+    pub fn get_costs(&mut self, output: &mut Vec<String>, cmd: &str) {
+        let snippet = cmd.to_string().split_off("::get_costs ".len());
+        self.run_snippet(output, true, &snippet.to_string());
+    }
```

Comment:
Looks like the two versions of `get_costs` are identical now, so we can merge into one and remove the feature gating.